exercises/gigasecond/test/test_gigasecond.c

```diff
@@ -0,0 +1,67 @@
+#include <time.h>
+
+#include "vendor/unity.h"
+#include "../src/gigasecond.h"
+
+/* Convert a date string in the format "YYYY-MM-DD HH:MM:SS +NNNN"
+ * (date, time, offset from UTC) into a time_t (seconds since Unix
+ * epoch).
+ */
+time_t parse_date(const char *const str) {
+  struct tm date;
+  strptime(str, "%F %T %z", &date);
+  return mktime(&date);
+}
+
+void test_date(void) {
```

Comment:
This problem has a [common test list](https://github.com/exercism/x-common/blob/master/gigasecond.json). Can you update these tests to match the common test list so that we can keep this exercise in sync with other tracks?